### PR TITLE
update length verbiage to differentiate behavior between BLOB bytes and CLOB characters

### PR DESCRIPTION
**Pull request by ashnazg**
This updates the doc verbiage based on @cjbj 's comments on https://bugs.php.net/bug.php?id=70700.

### CODE REVIEW

File: reference/oci8/ocilob/read.xml [hunk 1]
```diff
@@ -31,7 +32,8 @@
      <term><parameter>length</parameter></term>
      <listitem>
       <para>
-       The length of data to read, in bytes.  Large values will be rounded down to 1 MB.
+       The length of data to read, in bytes (BLOB) or characters (CLOB).
+       Large values will be rounded down to 1 MB.
```
Comment (ashnazg) [inline reference/oci8/ocilob/read.xml:36]: @cjbj , does this line about `rounded down to 1 MB` need to be adjusted based on the byte/character difference?

Comment (cjbj) [reply reference/oci8/ocilob/read.xml:36]: I expect so but would need to test.

Comment (ashnazg) [reply reference/oci8/ocilob/read.xml:36]: So then, is it reasonable to merge this PR, and I'll _separately_ keep that `1 MB` conversation going?

Comment (Girgias) [reply reference/oci8/ocilob/read.xml:36]: Merged for the time being.

**Pull request merged**